src/ConnectedMode/ConnectedMode.csproj

```diff
@@ -1,13 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
   
   <Import Project="..\SonarLint.Product.props" />
+  <Import Project="..\SonarLint.VsPkgRefs.props" />
 
   <PropertyGroup>
     <ProjectGuid>0BE551DB-3C46-42A5-BB38-DA80E83F8ABD</ProjectGuid>
     <AssemblyName>SonarLint.VisualStudio.ConnectedMode</AssemblyName>
     <RootNamespace>SonarLint.VisualStudio.ConnectedMode</RootNamespace>
 
     <RequiresLibGit>true</RequiresLibGit>
+
+    <LangVersion>latest</LangVersion>
```

Comment:
This is a bit problematic. We've been avoiding using it since it's possible that some of the changes are not syntactic sugar, which will then break on the client's machine. Is this for the `using namespace`? If so, let's revert it as it's not worth the risk (sadly).

Reply:
I will try to set it to 7 and see if it compiles. Maybe I needed it for some stuff that I ended up not using

Reply:
See Duncan's comment regarding this: https://github.com/SonarSource/sonarlint-visualstudio/pull/3576#discussion_r1091735735

Reply:
Yes, but that's why I changed to latest. Did I misunderstood him then?

Reply:
Yes, he meant that we shouldn't set this at all (only in test projects).